src/iosrv/srv.c

```diff
@@ -233,6 +233,122 @@ dss_sched_unit_pop(ABT_pool *pools, ABT_pool *pool)
 	return ABT_UNIT_NULL;
 }
 
+static struct dss_xstream *
+dss_xstream_get(int stream_id)
+{
+	if (stream_id == DSS_XS_SELF)
+		return dss_get_module_info()->dmi_xstream;
+
+	D_ASSERTF(stream_id >= 0 && stream_id < xstream_data.xd_xs_nr,
+		  "invalid stream id %d (xstream_data.xd_xs_nr %d).\n",
+		  stream_id, xstream_data.xd_xs_nr);
+
+	return xstream_data.xd_xs_ptrs[stream_id];
+}
+
+/* Add to the sorted(by expire time) list */
+static void
+dss_add_expire_list(struct dss_xstream *dx, struct dss_timeout_ult *new)
```

Comment:
remove 'dss' prefix for internal function.

Reply:
ok